public/data/avie-dev.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Avie Fukai",
+  "type": "personal",
+  "bio": "A Filipino SE in Japan",
+  "avatar": "https://github.com/avie-dev.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/avie-dev",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AvieDev",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "www.linkedin.com/in/jfukai",
```

Comment:
Best practice is to use the complete syntax for the URL to avoid some errors and warnings in the console while loading! 

```suggestion
      "url": "https://www.linkedin.com/in/jfukai",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix